install/0.2.1/kfserving.yaml

```diff
@@ -782,7 +782,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: proxy-role
+  name: kfserving-proxy-role
```

Comment:
We shouldn't be changing these releases.

Reply:
proxy-role does not exist, I think this is safe to change, it is already fixed in kubeflow manifest. We need a better procedure to sync between kubeflow manifests and here.